examples/FileUpload.custom.tsx

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import {
+  FileUpload,
+  FileUploadCard,
+  FileEmptyCard,
+  IconButton,
+} from '@itwin/itwinui-react';
+import {
+  SvgSmileySadVery,
+  SvgSmileyHappyVery,
+  SvgClose,
+} from '@itwin/itwinui-icons-react';
+
+export default () => {
+  const inputRef = React.useRef<HTMLInputElement>(null);
+  const [files, setFiles] = React.useState<File[]>([]);
+  const emptyCard = (
+    <FileEmptyCard>
+      <FileEmptyCard.Icon>
+        <SvgSmileySadVery />
+      </FileEmptyCard.Icon>
+      <FileEmptyCard.Text>
+        <FileUploadCard.InputLabel>Custom Label Text</FileUploadCard.InputLabel>
+        <div>Custom Description Text</div>
+      </FileEmptyCard.Text>
+    </FileEmptyCard>
+  );
+  return (
+    <FileUpload
+      onFileDropped={(files) => {
+        setFiles(files);
```

Comment:
There is some type error here.

![image](https://github.com/iTwin/iTwinUI/assets/45748283/f46f854b-9d05-47ab-84cd-121af1931f8a)